Source/Fuse.Controls.Panels/Layouts/WrapLayout.uno

```diff
@@ -135,6 +153,8 @@ namespace Fuse.Layouts
 			var placements = new float4[elements.Count];
 			//minorMaxSize in each major row, assinged per element
 			var minorSizes = new float[elements.Count];
+			// save the available space for each row
+			var majorRest = new Dictionary<float, float>();
```

Comment:
Yes. So the alternative to the solution I made is to have an int array of which row each element is on, and then a float array of majorRest per row. The current implementation is easier, but this one might be easier to understand. I'm still not convinced that is possible to miss on the `Dictionary` though, without a repro.